lib/jobs/install-os.js

```diff
@@ -96,6 +96,23 @@ function installOsJobFactory(
                 }
             });
         }
+
+        if (this.options.installPartitions) {
+            _.forEach(this.options.installPartitions, function(partition) {
+                assert.string(partition.mountPoint);
+                assert.string(partition.size);
```

Comment:
As I commented in the doc PR (https://github.com/RackHD/docs/pull/278), the `size` should be better to be a number.